phases/phase-1/challenges/posts/UbikCapital

```diff
@@ -0,0 +1 @@
+A5CDAC5623F6BEE4409CF2EAB8003AA6170C49A652198C9804FD0DFACE6994C8
```

Comment:
@UbikCapital This looks like a reaction transaction. Would you mind replacing it with a post one?

Reply:
```suggestion
E7D3F8F1A01E64B185C0D8230BB1170FB373DC2FC2673A3A13AF61DB681EA92F
```

This should do it